modules/fastp/main.nf

```diff
@@ -29,13 +29,12 @@ process FASTP {
     // Added soft-links to original fastqs for consistent naming in MultiQC
     def prefix = task.ext.prefix ?: "${meta.id}"
     // Use single ended for interleaved. Add --interleaved_in in config.
-    if (meta.single_end) {
+    if ( task.ext.args?.contains('--interleaved_in') ) {
         def fail_fastq = save_trimmed_fail ? "--failed_out ${prefix}.fail.fastq.gz" : ''
```

Comment:
Nitpicking, but can we move this line outside `if` to avoid duplicate code?

Reply:
Went one further (the separate variables was legacy from the earlier version)

Reply:
❤️